### PR TITLE
#158755841 Onboard customers to the platform

### PR DESCRIPTION
**Pull request by Rhotimee**
#### What does this PR do?
- Implement onboarding for customers

#### Description of Task to be completed?
- Make take a tour a drop-down
- Implement take a tour feature

#### How should this be manually tested?
- Setup application with instructions in the readme
- Click on take a tour

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#158755841](https://www.pivotaltracker.com/story/show/158755841)

#### Snapshot
screen shot 2018-07-29 at 9 14 53 am
screen shot 2018-07-29 at 9 16 36 am
screen shot 2018-07-29 at 9 16 54 am

### CODE REVIEW

**Comment by Daramola98** [comment]
@Rhotimee Good job but why do we have 3 commits for this feature?

**Comment by saheedt** [comment]
@Rhotimee please squash your commits into one

**Comment by Rhotimee** [comment]
@saheedt & @Daramola98, I'd do that.

**Comment by ddouglasz** [comment]
Asides squashing your commits, LGTM. 🧐

**Comment by OlatunjiYso** [comment]
@Rhotimee ,
can you have your story id point to the pivotal tracker ?
such that it would be clickable to reveal the story.
`[Story Id comes here ]( url comes here )` and it would become a link.

**Comment by Rhotimee** [comment]
@OlatunjiYso I've done that. Thanks

File: imports/plugins/core/ui-navbar/client/components/navbar.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Components } from "@reactioncommerce/reaction-components";
 import { Meteor } from "meteor/meteor";
 import { Reaction } from "/client/api";
 import startTour from "/imports/plugins/custom/vendor-tour/client/vendorTour";
+import StartCustomerTour from "/imports/plugins/custom/customer-tour/client/customerTour";
```
Comment (clintfidel) [inline imports/plugins/core/ui-navbar/client/components/navbar.js]: @Rhotimee, good work, however I see that  StartCustomerTour is in pascal case, please let's be consistent on our naming conventions.

**Review by clintfidel — DISMISSED** [review]
@Rhotimee , good job, please check my comment on this PR thanks.

**Comment by abdulfataiaka** [comment]
@Rhotimee, please ensure to address @clintfidel's comment and provide snapshots of what you have done in your PR description 🙇

**Comment by Daramola98** [comment]
LGTM

**Pull request merged**